### PR TITLE
INS-3686: addMigrationAddress spec api validation

### PR DESCRIPTION
**Pull request by sexyhamster**
**- What I did**
updated internal api spec according to 
https://github.com/insolar/insolar-api/pull/39

### CODE REVIEW

File: application/appfoundation/helpers.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func IsMigrationDaemonMember(member insolar.Reference) bool {
 	return false
 }
 
-var etheriumAddressRegex = regexp.MustCompile(`^(0x)?[\dA-Fa-f]{40}$`)
+var etheriumAddressRegex = regexp.MustCompile(`^(0x)?[\dA-Fa-f]{64}$`)
```
Comment (ita-sammann) [inline application/appfoundation/helpers.go]: WAT?

Comment (sexyhamster) [reply application/appfoundation/helpers.go]: oops

File: application/api/spec/api-exported.yaml [hunk 2]
```diff
@@ -1987,6 +1987,7 @@ components:
       x-json-schema-id: response/daemon-status.yaml
     migrationAddress:
       type: string
+      pattern: '^0x[a-f0-9]{40}$'
```
Comment (ita-sammann) [inline application/api/spec/api-exported.yaml]: Regex doe

Comment (ita-sammann) [inline application/api/spec/api-exported.yaml]: Regex confilcts with example. Doesn't contain capital A-F

**Pull request merged**